packages/nextjs/app/builders/0x7f6Ba565e35BB23C307992feBD841D5b2d8c8011/page.tsx

```diff
@@ -0,0 +1,58 @@
+import Link from "next/link";
+import type { NextPage } from "next";
+import { CogIcon, LinkIcon } from "@heroicons/react/24/outline";
+import { UserIcon } from "@heroicons/react/24/outline";
+import { Address } from "~~/components/scaffold-eth";
+
+const Home: NextPage = () => {
```

Comment:
Please name your page with something more unique, like your name or nickname.

Reply:
@fjkang I mean the component name. It's `Home`. Please, change to something unique.

Reply:
@fjkang I mean the component name. It's Home. Please, change to something unique.